pkg/controller/utils/machine_helper.go

```diff
@@ -0,0 +1,111 @@
+package utils
+
+import (
+	"context"
+	"reflect"
+	"strings"
+
+	machineapi "github.com/openshift/machine-api-operator/pkg/apis/machine/v1beta1"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+
+	awsproviderapi "sigs.k8s.io/cluster-api-provider-aws/pkg/apis/awsproviderconfig/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var log = logf.Log.WithName("machine_helper")
+
+func RemoveLBFromMasterMachines(kclient client.Client, elbName string, masterNodes *machineapi.MachineList) error {
```

Comment:
Could you add a comment for this function? It should look like:

```
// RemoveLBFromMasterMachines ....
```

Check out [this blog post](https://blog.golang.org/godoc-documenting-go-code) for more on how Go wants code documented.